packages/x-date-pickers-pro/README.md

```diff
@@ -42,15 +42,6 @@ This component has the following peer dependencies that you will need to install
 },
 ```
 
-If you need to use `js-joda`, `date-fns-jalali`, `jalaali`, or `hijri` library, you should be able to find the corresponding date-library from [`@date-io`](https://github.com/dmtrKovalenko/date-io#projects).
```

Comment:
We need to rework those README files...

Reply:
Oh yeah, they could use some touch-up or even a DevEx cc @samuelsycamore eye, as they are somewhat visible, especially in npm.js and GH. 🤔